esp/esp/dbmail/models.py

```diff
@@ -345,7 +348,7 @@ def process(self, debug=False):
                 # all recipients the first time.
                 newtxt, created = TextOfEmail.objects.get_or_create(**newtxt)
                 if not created:
-                    if debug: print 'Skipped duplicate creation of message to %s for message request %d: %s' % (send_to, self.id, self.subject)
+                    logger.warning('Skipped duplicate creation of message to %s for message request %d: %s', send_to, self.id, self.subject)
```

Comment:
It looks like it prints a unicode error, but continues on, which is also not the most ideal.  In any case I can just make the actual fix while I'm at it.